docs/HumanHelp.md

```diff
@@ -0,0 +1,56 @@
+# Help, I can't find an issue template that gets me what I need! What do I do now?!
```

Comment:
this should not be an h1 since it it just a problem/solution

we should have a top level h1 that outlines what this document is about as well as a short summary